integration_tests/src/util/mod.rs

```diff
@@ -42,6 +42,9 @@ pub fn create_accounts_seeded(
     seed: u64,
     test_faucet_addr: &Address,
 ) -> Vec<Address> {
+    println!("test_faucet_addr {:?}", test_faucet_addr);
+    let faucet = v.actor(test_faucet_addr).unwrap();
+    println!("faucet: {:?}", faucet);
```

Comment:
When using TEST_FAUCET_ADDR, we will get the correct non-zero faucet.balance here. 

However, the below send will fail in the FVM itself which claims 0 balance for the same address. (when run on anorth/fvm-workbench).

Reply:
But it all works when you use 98? Do other random addresses work too, or is that one special?

Reply:
Init actor state was being populated with `state.next_id` as FIRST_NON_SINGLETON_ADDR. 

So accounts were being created at id 100 -> 102 which overlapped with the existing hardcoded TEST_FAUCET_ADDR location.